src/conversion/model-to-view-converters.js

```diff
@@ -108,107 +135,95 @@ export function insertUIElement( elementCreator ) {
 	return ( evt, data, consumable, conversionApi ) => {
 		let viewStartElement, viewEndElement;
 
+		// Create two view elements. One will be inserted at the beginning of marker, one at the end.
+		// If marker is collapsed, only "opening" element will be inserted.
 		if ( elementCreator instanceof ViewElement ) {
 			viewStartElement = elementCreator.clone( true );
 			viewEndElement = elementCreator.clone( true );
 		} else {
 			data.isOpening = true;
-			viewStartElement = elementCreator( data, consumable, conversionApi );
+			viewStartElement = elementCreator( data, conversionApi );
 
 			data.isOpening = false;
-			viewEndElement = elementCreator( data, consumable, conversionApi );
+			viewEndElement = elementCreator( data, conversionApi );
 		}
 
 		if ( !viewStartElement || !viewEndElement ) {
 			return;
 		}
 
 		const markerRange = data.markerRange;
-		const eventName = evt.name;
-
-		// Marker that is collapsed has consumable build differently that non-collapsed one.
-		// For more information see `addMarker` and `removeMarker` events description.
-		// If marker's range is collapsed - check if it can be consumed.
-		if ( markerRange.isCollapsed && !consumable.consume( markerRange, eventName ) ) {
-			return;
-		}
-
-		// If marker's range is not collapsed - consume all items inside.
-		for ( const value of markerRange ) {
-			if ( !consumable.consume( value.item, eventName ) ) {
```

Comment:
Why consume is removed?

Reply:
For some time there were no consumables and I forgot to bring back this part of code.